mitxpro/settings.py

```diff
@@ -32,6 +32,13 @@
 
 SECURE_SSL_REDIRECT = get_bool("MITXPRO_SECURE_SSL_REDIRECT", True)
 
+HUBSPOT_CONFIG = {
+    "NEW_COURSES_FORM_GUID": get_string(
+        "NEW_COURSES_FORM_GUID", "b9220dc1-4e48-4097-8539-9f2907f18b1e"
+    ),
+    "PORTAL_ID": get_string("PORTAL_ID", "5890463"),
+}
+
```

Comment:
@pdpinch @rhysyngsun Shouldn't those be specified in `app.json` and if they're going into heroku config vars and that's a flat file, shouldn't the keys be more descriptive (ex. HUBSPOT_PORTAL_ID)

Reply:
Yes to both